kpi/serializers/current_user.py

```diff
@@ -106,9 +106,10 @@ def update(self, instance, validated_data):
                         'current_password': 'Incorrect current password.'
                     })
         elif any((current_password, new_password)):
-            raise serializers.ValidationError(
-                'current_password and new_password must both be sent ' \
-                'together; one or the other cannot be sent individually.'
-            )
+            raise serializers.ValidationError({
+                'password': 'current_password and new_password must both be '
```

Comment:
unless i am not remembering the DRF documentation correctly, keys in this dictionary should be the names of fields. that's not really a perfect fit for this case, because the error deals with two fields at once, but i guess the lesser evil is to pick one field or the other to serve as the dict key

Reply:
or move it to a validate method as you mentioned, @noliveleger?